cypress/integration/mobile/landscape/visitorCanSearchForServices.feature.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-undef */
```

Comment:
This to remove compiler warnings or? what is going on?

Reply:
yeah it remove the warnings i get under ```cy``` so my test file si no all red :)